Authorization/Authorization/Presentation/Base/FieldsView.swift

```diff
@@ -94,8 +95,8 @@ struct FieldsView: View {
                 }
             }
             Text(.init(text))
-                .tint(Theme.Colors.infoColor)
-                .foregroundStyle(Theme.Colors.textSecondaryLight)
+                .tint(themeManager.theme.colors.infoColor)
+                .foregroundStyle(themeManager.theme.colors.textSecondaryLight)
```

Comment:
I think we should avoid adding uneeded diff, so the Theme class should be updated at runtime in-place:

```suggestion
                .tint(Theme.colors.infoColor)
                .foregroundStyle(Theme.colors.textSecondaryLight)
```